src/index.js

```diff
@@ -0,0 +1,85 @@
+// Importing dependancies
+const inquirer = require("inquirer");
+const path = require("path");
+const fs = require("fs");
+const { writeToFile } = require("./utils/utils.js");
+const generateHTMLfile = require("./utils/renderHtml.js");
+
+// Importing Classes
+const { name, id, email } = require("./lib/Employee");
```

Comment:
This is an unnecessary import - firstly these values aren't exported from this file, and you're also not using them anywhere.